CIScripts/Test/Utils/IisTcpTestDockerImageBuild.Tests.ps1

```diff
@@ -0,0 +1,24 @@
+Param (
+    [Parameter(Mandatory=$true)] [string] $TestenvConfFile
+)
+
+. $PSScriptRoot\IisTcpTestDockerImageBuild.ps1
+
+$Sessions = New-RemoteSessions -VMs (Read-TestbedsConfig -Path $TestenvConfFile)
+$Session = $Sessions[0]
+
+Describe "Initialize-IisTcpTestDockerImage" {
+    It "Builds iis-tcptest image" {
+        { Initialize-IisTcpTestDockerImage -Session $Session } | Should Not Throw
+
+        Invoke-Command -Session $Session {
+            docker inspect iis-tcptest
+        } | Should Not BeNullOrEmpty
+    }
+
+    BeforeEach {
+        Invoke-Command -Session $Session {
+            docker image rm iis-tcptest -f
```

Comment:
This line seems to fail if the image is not present. To silence the error you'd need to wrap it in a block with `SilentlyContinue` and add stderr redirection (note to self: we should allow invoking commands on remote session in `Invoke-NativeCommand`):
```
        Invoke-Command -Session $Session {
            Invoke-Command {
                $ErrorActionPreference = 'SilentlyContinue'
                docker image rm iis-tcptest -f 2>$null
            }
        }
```